Package.swift

```diff
@@ -43,8 +43,14 @@ let package = Package(
         .library(
             name: "Web3Inbox",
             targets: ["Web3Inbox"]),
+        .library(
+            name: "Web3Modal",
+            targets: ["Web3Modal"]),
+
+    ],
+    dependencies: [
+        .package(url: "https://github.com/dagronf/QRCode", from: "13.9.0")
```

Comment:
I would fork it right now, it's too much risk

Reply:
Sure. On it.